src/log/appender.cpp

```diff
@@ -1,15 +1,13 @@
 #include <fc/log/appender.hpp>
 #include <fc/log/console_appender.hpp>
 #include <fc/log/gelf_appender.hpp>
-#include <fc/log/dmlog_appender.hpp>
 #include <fc/log/logger_config.hpp>
 
 
 namespace fc {
 
    static bool reg_console_appender = log_config::register_appender<console_appender>( "console" );
    static bool reg_gelf_appender = log_config::register_appender<gelf_appender>( "gelf" );
```

Comment:
These other two appenders are no longer used, correct? They should be removed. I don't think we have a requirement to support gelf logging anymore. I think Santhosh is looking to verify that.

Reply:
Correct. Appenders stdout/stderr/gelf have been removed.